src/schema/me/bidder_position.js

```diff
@@ -17,5 +24,35 @@ export const BidderPosition = {
   ) =>
     meBidderPositionLoader({
       id,
-    }).then(response => response.body),
+    }).then(response => {
+      const position = response.body
+      let status
+      if (anyReserveMet(position) && position.processed_at && position.active) {
+        return {
+          status: "SUCCESS",
```

Comment:
The naming here is a little confusing to me-- even though we show the error-looking symbol on the emission views, I think _all_ cases where the bidder position is processed are technically "successful", and there's only really one true error case.

Kind of hard to describe-- but maybe something like this maps a little better?
```js
if (position.processed_at) {
  if (!position.active) {
    status = 'OUTBID'
  } else if (anyReserveMet(position)) {
    status = 'WINNING'
  } else {
    status = 'RESERVE_NOT_MET'
  }
} else if (!position.processed_at) {
  status = 'PENDING'
} else {
  status = 'ERROR'
}
```